$OpenBSD: patch-kde3support_kunittest_CMakeLists_txt,v 1.1 2013/08/12 01:29:03 zhuk Exp $
Deconflict with KDE3.
Will require patching any KDE4 software that uses kunittestmodrunner, but
there should be not that much now.
--- kde3support/kunittest/CMakeLists.txt.orig	Sat Jul 27 11:43:27 2013
+++ kde3support/kunittest/CMakeLists.txt	Sat Jul 27 11:45:39 2013
@@ -27,7 +27,7 @@ kde4_add_executable(kunittestmodrunner NOGUI ${kunitte
 
 target_link_libraries(kunittestmodrunner ${KDE4_KDECORE_LIBS} kunittest)
 
-install(TARGETS kunittestmodrunner DESTINATION ${BIN_INSTALL_DIR} )
+install(TARGETS kunittestmodrunner DESTINATION ${BIN_INSTALL_DIR} RENAME kunittestmodrunner4)
 
 
 ########### install files ###############
